backend/app/assets/stylesheets/spree/backend/components/_navigation.scss

```
@@ -355,8 +355,11 @@ nav.menu {
 
 .brand-link {
   display: block;
+  height: $main-header-height - 1;
```

Might be worth giving the `1` a unit. I imagine other things might break if someone overrode the height with a `rem` value, but we might not as well not add to that.

```suggestion
  height: $main-header-height - 1px;
```